VideoWeb/VideoWeb/ClientApp/src/app/testing/mocks/mock-consultation-service.ts

```diff
@@ -14,14 +14,18 @@ export function consultationServiceSpyFactory(): jasmine.SpyObj<ConsultationServ
         'respondToAdminConsultationRequest',
         'clearModals',
         'startPrivateConsulationWithEndpoint',
-        'cancelTimedOutIncomingRequest'
+        'cancelTimedOutIncomingRequest',
```

Comment:
should there be a mock for this?

Reply:
Yup, pre-existing tests use this. see `wr-participant-list-shared.component.spec.ts`